### PR TITLE
Release Notes for November 2024

### PR DESCRIPTION
**Pull request by mikehoffms**
Article for review:
* **Release Notes for the WebView2 SDK** > **1.0.2903.40**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3294#10290340
   * [Planned live](https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#10290340)
* **Release Notes for the WebView2 SDK** > **1.0.2950-prerelease**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3294#102950-prerelease
   * [Planned live](https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#102950-prerelease)

AB#54657173

### CODE REVIEW

File: microsoft-edge/webview2/release-notes/index.md [hunk 1]
```diff
@@ -6,121 +6,182 @@ ms.author: msedgedevrel
...
 ###### heading
 
 description
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
+* [CoreWebView2NonClientRegionKind.Close Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2nonclientregionkind?view=webview2-dotnet-1.0.2950-prerelease&preserve-view=true)
+* [CoreWebView2NonClientRegionKind.Maximize Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2nonclientregionkind?view=webview2-dotnet-1.0.2950-prerelease&preserve-view=true)
+* [CoreWebView2NonClientRegionKind.Minimize Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2nonclientregionkind?view=webview2-dotnet-1.0.2950-prerelease&preserve-view=true)
+
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/release-notes/index.md:137]: @sivMSFT do remove the above

Comment (sivMSFT) [reply microsoft-edge/webview2/release-notes/index.md]: Updated @victorhuangwq & @mikehoffms

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/index.md]: confirmed gone from Prerel section

File: microsoft-edge/webview2/release-notes/index.md [hunk 2]
```diff
@@ -97,33 +95,200 @@ description
...
+###### Control whether the screen capture UI is shown (ScreenCaptureStarting event)
+
+Added a new ScreenCaptureStarting event.  This event is raised whenever the WebView2 and/or iframe that corresponds to the CoreWebView2Frame (or to any of its descendant iframes) requests permission to use the Screen Capture API before the UI is shown.  The app can then block the UI from being displayed, or allow the UI to be displayed.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- todo: global: un-link parent if had members already -->
+
+* `CoreWebView2` Class:
+   * [CoreWebView2.Cancel Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.cancel?view=webview2-dotnet-1.0.2903.40&preserve-view=true)<!-- todo: copied from rt, ok? -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/index.md:148]: I added these 4 properties b/c they are in WinRT tab.  ok?
Build report claims 404 only for these 4 Property members that i manually constructed.
ScreenCaptureStarting Event seems ok.

@sivMSFT @victorhuangwq

File: microsoft-edge/webview2/release-notes/index.md [hunk 3]
```diff
@@ -97,33 +95,190 @@ description
...
+* `CoreWebView2Frame` Class:
+   * [CoreWebView2Frame.ScreenCaptureStarting Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view=webview2-winrt-1.0.2903.40&preserve-view=true#screencapturestarting)
+
+* [CoreWebView2NonClientRegionKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2nonclientregionkind?view=webview2-winrt-1.0.2903.40&preserve-view=true)
+   * `CoreWebView2NonClientRegionKind.Close`
+   * `CoreWebView2NonClientRegionKind.Maximize`
+   * `CoreWebView2NonClientRegionKind.Minimize`
+
+* [CoreWebView2ScreenCaptureStartingEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2screencapturestartingeventargs?view=webview2-winrt-1.0.2903.40&preserve-view=true)
+   * [CoreWebView2ScreenCaptureStartingEventArgs.Cancel Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2screencapturestartingeventargs?view=webview2-winrt-1.0.2903.40&preserve-view=true)<!-- todo: add member to url, copied from .net - ok? -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/index.md:190]: todo: after WinRT Ref 2903 is live, test WinRT links

Presently WinRT Ref has no **Version** menu in upper left:
https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/
F12 > right-click "Refresh" > "Empty cache and hard refresh" > F12 (loads latest .png files & TOC)

**Review by peiche-jessica — APPROVED** [review]
LGTM.

**Comment by sivMSFT** [comment]
LGTM - PM

**Review by sivMSFT — APPROVED** [review]
LGTM - PM

**Pull request merged**